+php-laravel-mode/.yas-parents.el

```diff
@@ -0,0 +1,26 @@
+(eval-when-compile (require 'subr-x))
+
+(defun string-split-words (string)
+  (split-string
+   (let ((case-fold-search nil))
+     (replace-regexp-in-string
+      "\\([[:lower:]]\\)\\([[:upper:]]\\)" "\\1 \\2"
+      (replace-regexp-in-string "\\([[:upper:]]\\)\\([[:upper:]][0-9[:lower:]]\\)" "\\1 \\2" string)))
+   "[^[:word:]0-9]+"
+   t))
+
+(defun +php-laravel-mode--get-namespace ()
+    "Get a formatted namespace for the current PHP file"
+    (substring
+        (replace-regexp-in-string "/" (regexp-quote "\\")
+            (thread-first
+                buffer-file-name
+                (file-relative-name (doom-project-root))
+                file-name-directory
+                capitalize))
+        0
+        -1))
+
+(defun +php-laravel-mode--get-class-name ()
+    "Get a formatted class name for the current PHP file"
+    (string-join (mapcar 'capitalize (string-split-words (file-name-base buffer-file-name)))))
```

Comment:
Please reindent (should be 2 spaces per level, not 4)

Reply:
That's how indent region is doing it now for some reason. Will try and edit manually